src/Files.App/Utils/StatusCenter/StatusCenterItem.cs

```diff
@@ -199,7 +201,7 @@ public StatusCenterItem(
 			AnimatedIconState = "NormalOff";
 			SpeedGraphValues = [];
 			CancelCommand = new RelayCommand(ExecuteCancelCommand);
-			Message = Strings.ProcessingItems.GetLocalizedResource();
```

Comment:
Is this string being used anywhere else? If not, it can be removed from the resource file.